### PR TITLE
IDNSEIT-862 Update Selenium-grid-scaler module to latest selenium and aws-java sdk

### PR DESCRIPTION
**Pull request by navaneeth-basavaraj-SP**

### CODE REVIEW

File: selenium-grid-scaler/src/main/java/com/rmn/qa/AutomationRunRequest.java [hunk 1]
```diff
@@ -228,29 +228,57 @@ public String toString() {
...
         }
+
+        if(capabilityPlatformObject == null) {
+            capabilityPlatformObject = capabilities.get(CapabilityType.PLATFORM);
+        }
         Platform capabilityPlatform = AutomationUtils.getPlatformFromObject(capabilityPlatformObject);
-        if(!AutomationUtils.lowerCaseMatch(browser, capabilityBrowser)) {
+
+        if(!capabilities.containsValue(browser)) {
+            log.info("Returning here; browser " + browser + " & " + capabilityBrowser);
```
Comment (carlos-urrutia-sp) [inline selenium-grid-scaler/src/main/java/com/rmn/qa/AutomationRunRequest.java]: "Returning here" log seems to broad, can we add something like didn't know match browser instead?

File: selenium-grid-scaler/src/main/java/com/rmn/qa/AutomationRunRequest.java [hunk 2]
```diff
@@ -228,29 +228,57 @@ public String toString() {
...
             return false;
         }
         if(browserVersion != null && !AutomationUtils.lowerCaseMatch(browserVersion,capabilityBrowserVersion))  {
+            log.info("Returning here; browser version " + browserVersion + " & " + capabilityBrowserVersion);
```
Comment (carlos-urrutia-sp) [inline selenium-grid-scaler/src/main/java/com/rmn/qa/AutomationRunRequest.java]: same thing here

File: selenium-grid-scaler/src/main/java/com/rmn/qa/AutomationRunRequest.java [hunk 3]
```diff
@@ -228,29 +228,57 @@ public String toString() {
...
             return false;
         }
         if(platform != null && !AutomationUtils.firstPlatformCanBeFulfilledBySecondPlatform(platform, capabilityPlatform)) {
+            log.info("Returning here; platform version: " + platform + " & " + capabilityPlatform);
```
Comment (carlos-urrutia-sp) [inline selenium-grid-scaler/src/main/java/com/rmn/qa/AutomationRunRequest.java]: same here somethinkg like did not match platform

File: selenium-grid-scaler/src/main/java/com/rmn/qa/AutomationRunRequest.java (hunk 1 above)
Comment (navaneeth-basavaraj-SP) [reply selenium-grid-scaler/src/main/java/com/rmn/qa/AutomationRunRequest.java]: I'll remove those, added for debugging purpose.

**Pull request merged**